Use adopt@1.11.0-4 in TravisCI

ignasi35 opened the pull request:
Fixes #2197

dwijnand commented:
I suggest we change Travis CI to use the latest patched version intrinsically.  See https://github.com/playframework/play-meta/issues/93.

ignasi35 commented:
> PR title says `4`. Change says `3`. `1.11.0-4` exist. Did you mean to use it instead of `1.11.0-3`?

🤦🏼‍♂️No, I meant to use `.4`. Started out thinking `.3`, then went on a yak shave and thought I had changed it here too when I sent the PR. Fixed and force pushed.
